lib/controller.js

```diff
@@ -92,7 +95,7 @@ Tessel.list = function(opts) {
 
 Tessel.get = function(opts) {
   return new Promise(function(resolve, reject) {
-    logs.info('Looking for your Tessel...');
+    logs.info('Searching nearby LAN (netmask) and USB for Tessels...');
```

Comment:
@Student007 I get where you are coming from but 90% of users won't care what types of connections the CLI is looking for. They will just want to deploy their code and it should work regardless of the connection type. Let's remove it from the PR because it's not related to `t2 root` and if you feel strongly about it, let's open an issue so more people can discuss. :)